platform/client/src/pages/ViewActionPage.tsx

```diff
@@ -85,7 +85,18 @@ export const ViewActionPage: React.FC<RouteComponentProps<IUpdateMatchParams>> =
                 <AutoFields />
                 <ErrorsField />
               </AutoForm>
-              <Map lat={model.distributionCentre?.lat!} long={model.distributionCentre?.long!} name={model.distributionCentre?.name!}></Map>
+              <Map center={{
+                    lat: parseInt(model.distributionCentre?.lat!),
+                    lng: parseInt(model.distributionCentre?.long!)
+                  }}>
+                <Marker
+                  label={model.distributionCentre?.name!}
+                  title="The marker`s title will appear as a tooltip"
```

Comment:
```suggestion
                  title=`{model.distributionCentre?.address1}
{model.distributionCentre?.address2}
{model.distributionCentre?.city}`
                  
```

Reply:
Actually it will be nicer to build them beforehand and just use them here :D

Reply:
Ah good catch.  I like the suggestion also.